src/pages/builders.js

```diff
@@ -219,7 +219,17 @@ const DeveloperChampionPage = () => {
               <hr />
               <ul>
                 <h4>It is nice to have:</h4>
-                <li>Be an active member of The Reliquary,</li>
+                <li>
+                  Be an active member of{' '}
+                  <a
```

Comment:
Very minor nit:

Could you use the `Link` component from `@newrelic/gatsby-theme-newrelic` here? We just added the ability to track external page clicks via NR Browser and would love if this were instrumented as well. We get that for free with that component. Could you update to that component?

And just FYI, you can drastically simplify this component as `Link` will pre-fill the `target` and `rel` properties on the anchor tag. Use the following:

```js
import { Link } from '@newrelic/gatsby-theme-newrelic'

<Link to="https://therelicans.com">
  The Relicans
</Link>
```

Reply:
Sure! Thanks for the tip!